core/modules/nfv_ctrl.h

```diff
@@ -37,14 +38,19 @@ class NFVCtrl final : public Module {
 
  private:
   // All available per-core packet queues in a cluster
+  std::map<uint16_t, uint16_t> longTermOptimization(std::vector<double> flow_rate_per_bucket);
```

Comment:
longTermOptimization -> LongTermOptimization
Also, add a short comment on what the function does.